src/test/java/io/strimzi/kafka/bridge/http/base/HttpBridgeTestBase.java

```diff
@@ -154,19 +157,11 @@ static void afterAll(VertxTestContext context) {
         }
     }
 
-    @AfterEach
-    void tearDown() throws ExecutionException, InterruptedException {
+    @BeforeEach
+    void tearDown() throws ExecutionException, InterruptedException, TimeoutException {
+        topic = "my-topic-" + new Random().nextInt(Integer.MAX_VALUE);
```

Comment:
didn't we discuss on this? isn't it better having a meaningful name for each topic to understand more about a test failure?

Reply:
Hmmm, yes we do. I would rather use this generated topic on the topic. It is the same approach as we do in `strimzi-kafka-operator` repo.